generate/generate.go

```diff
@@ -571,6 +556,11 @@ func forEachStack(root string, callback forEachStackCallback) []error {
 			Str("stack", stackpath).
 			Logger()
 
+		if !strings.HasPrefix(stackpath, workingDir) {
+			logger.Trace().Msg("discarding stack outside working dir")
```

Comment:
```suggestion
			logger.Trace().
				Str("stackdir", stackpath).
				Msg("discarding stack outside working dir")
```
useful if troubleshooting why it's not calling the callback for some stack.

Reply:
The logger created on the loop already have the stack path:

```go
logger := logger.With().
			Str("stack", stackpath).
			Logger()
```

https://github.com/mineiros-io/terramate/pull/164/files#diff-39a24c8d3907e38beb5e39b5f0a40df7aa73c38449615f140ec3fca502900e86R556

Reply:
ah sorry, yeah.